TorchSharp/TorchSharp.csproj

```diff
@@ -44,9 +45,18 @@
     <Compile Remove="T4Template.cs" />
     <Compile Remove="Handle.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Numerics.Tensors" Version="0.1.0-preview2-181029-2" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
+  </ItemGroup>
   <ItemGroup>
     <Compile Update="TypeGeneration.cs">
       <DependentUpon>TypeGeneration.tt</DependentUpon>
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
What is this?

Reply:
No idea, will remove this (my VS is creating weird stuff).